app/src/main/java/app/revanced/integrations/patches/components/PlayerFlyoutMenuItemsFilter.java

```diff
@@ -0,0 +1,72 @@
+package app.revanced.integrations.patches.components;
+
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+
+import app.revanced.integrations.settings.SettingsEnum;
+import app.revanced.integrations.utils.ReVancedUtils;
+
+public class PlayerFlyoutMenuItemsFilter extends Filter {
+    private static String[] exceptions;
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    public PlayerFlyoutMenuItemsFilter() {
+        exceptions = new String[]{
```

Comment:
Please comment each exception and why they are required each in which situation if possible.